gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -90,6 +90,18 @@ export default function NxFormLayoutExample() {
           Blue
         </NxRadio>
       </fieldset>
+      <div className="nx-form-group">
+        <label className="nx-label">
+          <span className="nx-label__text">Select</span>
+          <select className="nx-form-select">
```

Comment:
This example should probably use the `select` as a controlled component (e.g. track its value with `useState` like we do for the text inputs).

Reply:
> This example should probably use the `select` as a controlled component (e.g. track its value with `useState` like we do for the text inputs).

This is probably wrong (or incomplete) but how's this? https://github.com/sonatype/sonatype-react-shared-components/pull/203/commits/2ce5822229084a320e3f375e6bb9a074b18b4279

Reply:
> This is probably wrong (or incomplete) but how's this? 2ce5822

Yeah that's not right.  What you want is a single `useState` that tracks the current value of the `select`.  The value of that state would be set on the `select` using its `value` prop, and the `select`'s `onChange` callback would be used to update the state.

Reply:
> > This is probably wrong (or incomplete) but how's this? [2ce5822](https://github.com/sonatype/sonatype-react-shared-components/commit/2ce5822229084a320e3f375e6bb9a074b18b4279)
> 
> Yeah that's not right. What you want is a single `useState` that tracks the current value of the `select`. The value of that state would be set on the `select` using its `value` prop, and the `select`'s `onChange` callback would be used to update the state.

https://github.com/sonatype/sonatype-react-shared-components/pull/203/commits/73a62067e8611c9cfb541ead62e8f413ca8e829d